### DISCUSSION TITLE
My ID Lock 150 stopped working after upgrading to v1.30.0

### DISCUSSION THREAD

**Discussion by GD-Dal** [Koenkk/zigbee2mqtt#16508] [Devices]
It now has status as Unsupported and Z2M throws an error saying:
Received message from unsupported device with Zigbee model 'ID Lock 150' and manufacturer name 'Datek'

What could be the reason for this?

**Comment by Koenkk** [comment]
This will be fixed in a hotfix release (probably tomorrow)
